packages/@react-spectrum/menu/docs/Menu.mdx

```diff
@@ -406,7 +405,7 @@ import {SubmenuTrigger} from '@react-spectrum/menu';
 
 You can define a recursive function to render the nested menu items dynamically.
 
-```tsx example
+```tsx example keepIndividualImports
```

Comment:
could we do this with jsdocs instead I wonder? then it'd be a little more obvious where updates need to happen
this also doesn't allow us to specify it on an individual import level, it's either all or nothing for the example

what if we could do
```
/**
 * @beta
 */
function SubmenuTrigger...
```

I guess the code to do the replacement is pretty simple, already has this flag, and there's only one instance of it, so probably not worth worry about

Reply:
potentially? I haven't entirely parsed through the MDXTransformer code but think that should be possible. I think another idea could be to expand the change I made to MDXTransformer here to accept an array of paths that we don't want transformed to be the mono package link.